community/best_practices/feature_documentation.md

```diff
@@ -0,0 +1,150 @@
+&#9634; &nbsp; [Database tables reference](#database-tables-reference) <br>
+&#9634; &nbsp; [GS1 schema reference](#gs1-schema-reference) <br>
+&#9634; &nbsp; [REST API Reference](#rest-api-reference) <br>
+&#9634; &nbsp; [Rust SDK documentation](#rust-sdk-documentation) <br>
+&#9634; &nbsp; [CLI Command Reference](#cli-command-reference) <br>
+&#9634; &nbsp; [Man pages for grid subcommands](#man-pages)
+
+Grid documentation is written in [GitHub Flavored
+Markdown](https://docs.github.com/en/github/writing-on-github/about-writing-and-formatting-on-github).
+
+Read on to learn what's needed for each item.
+
+### Feature Overview
+
+Provide a high-level description that is focused on business-level concepts and
+benefits. Some technical information is appropriate, but details belong in the
+procedure and reference material. Include links to the detailed information as
+appropriate.
+
+*Example*: [Splinter: Canopy Application
+Framework](https://www.splinter.dev/docs/0.4/concepts/canopy_application_framework.html)
+
+### How-to Procedure
+
+Write a procedure for each task associated with the feature, such as creating a
+new product or setting up Pike organizations, agents, and permissions.
+
+* Start with a introduction that explains the end goal for the procedure, where
+  it fits in the Grid ecosystem, and the intended audience for the procedure
+  (who usually does this task).
+
+* Include any assumptions (like requiring an existing circuit) and prerequisites
+  such as user keys, service IDs, or example YAML files that will be used in the
+  procedure.
+
+* Then list the steps for the task (such as creating and managing products with
+  command-line options and YAML schema files). Include example commands and
+  output for each step. For extra credit, add troubleshooting tips for common
+  problems.
+
+*Example*: [Running Grid on
+Splinter](https://grid.hyperledger.org/docs/0.1/grid_on_splinter.html)
+
+### Smart Contract Specification
+
+Provide a topic that describes the smart contract's objects, namespace, address
+format, and transaction types.
+
+*Examples*: [Grid Track and Trace
+  Specification](https://grid.hyperledger.org/docs/0.1/grid_track_and_trace_family_specification.html),
+  [Sawtooth Supply Chain Transaction Family
+  Specification](https://sawtooth.hyperledger.org/docs/supply-chain/nightly/master/family_specification.html)
+
+### Database Tables Reference
+
+Provide a reference topic that describes the Grid daemon's database schema,
+as defined in
+[grid/daemon/src/database/schema.rs](https://github.com/hyperledger/grid/blob/master/daemon/src/database/schema.rs).
+
+*Examples*: [Splinter: Biome Database
+Tables](https://www.splinter.dev/docs/0.4/concepts/biome_user_management.html#biome-database-tables),
+[Splinter: Gameroom Walkthrough](https://www.splinter.dev/docs/0.4/examples/gameroom/walkthrough/)
+(see Section I-2.8, "Gameroom daemons write notification to Gameroom database")
+
+### GS1 Schema Reference
+
```

Comment:
Not totally sure what this means. Are there any examples here?

Reply:
For product, this would be the list of supported GS1 fields provided "out of the box". For location, its a list of all the GS1 fields you can set on a location.

Reply:
@dplumb94 and @vaporos - Would it be clearer to call this document a "GS1 Field Reference" instead?